charts/intel-gpu-resource-driver/values.yaml

```diff
@@ -37,6 +35,13 @@ kubeletPlugin:
       effect: "NoSchedule"
   affinity: {}
 
+cdi:
+  staticPath: /etc/cdi
+  dynamicPath: /var/run/cdi
```

Comment:
My environment requires alternative paths for cdi, so I need to be able to configure the host paths